### PR TITLE
Feature/agregar peso mascota

### LINKED ISSUE #1: Agregar peso a la mascota
Como usuario, quiero poder agregar el peso de las mascotas y asegurarme de que el valor no sea menor a 0, para poder gestionar mejor la información de las mascotas.

# Criterios de Aceptación
- El campo peso debe aceptar valores numéricos.
- El valor del peso no debe ser menor a 0.

Tareas
- [x] Agregar el atributo peso a la clase `Mascota`.
- [x] Implementar validación para asegurar que el peso no sea menor a 0.
- [x] Actualizar la interfaz de usuario para permitir la entrada del peso.
- [x] Agregar mask para el input del peso
- [x] Crear prueba unitaria para validar el campo peso.
- [x] Crear prueba de integración 
- [x] Crear prueba end to end

### PR DESCRIPTION
**Pull request by germansalinas1994**
Esta PR añade la funcionalidad para registrar el peso de una mascota en el sistema. Se han realizado las siguientes modificaciones:

Se ha añadido un nuevo campo en el modelo de Mascota para almacenar el peso.
Se han actualizado las vistas para manejar la entrada y visualización del peso.
Se han añadido validaciones para asegurar que el peso ingresado sea razonable.
Se han creado tests unitarios y de integración para esta nueva funcionalidad.

### CODE REVIEW

**Review by Fedebravo12 — APPROVED** [review]

File: app/templates/base.html [hunk 1]
```diff
@@ -7,24 +7,19 @@
...
 <body data-bs-theme="dark">
     {% include "partials/navbar.html" %}
     <main class="mt-5">
         {% block main %}{% endblock %}
     </main>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.9.0/js/bootstrap-datepicker.min.js"></script>
 
-    <script>
```
Comment (Fedebravo12) [inline app/templates/base.html:20]: Se recomienda que se deje el script del document ready para consumir el calendario desde cualquier parte de la aplicación y no repetir código.

File: app/tests_integration.py [hunk 2]
```diff
@@ -94,3 +96,118 @@ def test_edit_user_with_valid_data(self):
...
+    def test_edit_pet_with_valid_data(self):
+        # Creación de una mascota con datos iniciales.
+        pet = Pet.objects.create(
+            name="Fido",
+            breed="Golden Retriever",
+            birthday="2015-01-01",
+            weight="10.50",
+        )
+
+        # Impresión de los datos iniciales para depuración.
```
Comment (Fedebravo12) [inline app/tests_integration.py:181]: Seria ideal quitar los print, ya que fueron usados como ayuda pero no aportan nada a los tests en si. Ademas, vuelven a los resultados menos legibles.

**Review by Fedebravo12 — APPROVED** [review]

File: app/tests_integration.py
Comment (Fedebravo12) [inline app/tests_integration.py:1]: Seria ideal quitar los print dentro de los tests, ya que fueron usados como ayuda pero no aportan nada a los tests en si. Ademas, vuelven a los resultados menos legibles.

File: app/templates/pets/form.html [hunk 3]
```diff
@@ -54,22 +54,65 @@ <h1>Mascota</h1>
...
+                    {% endif %}
+                </div>
                 <button class="btn btn-primary">Guardar</button>
             </form>
         </div>
     </div>
 </div>
+
+<script>
+    $(document).ready(function () {
```
Comment (Fedebravo12) [inline app/templates/pets/form.html:94]: En este caso, se debería quitar el document ready para dejarlo únicamente en base.html

File: app/templates/pets/form.html [hunk 4]
```diff
@@ -54,22 +54,65 @@ <h1>Mascota</h1>
...
+
+        $(".datepicker").datepicker({
+            format: "dd/mm/yyyy",
+            language: "es",
+            autoclose: true,
+            todayHighlight: true,
+        });
+
+        Inputmask({
```
Comment (Fedebravo12) [inline app/templates/pets/form.html:96]: Se podría agregar a la mask un suffix para indicar que el peso es en kilogramos.

**Review by Fedebravo12 — CHANGES_REQUESTED** [review]

**Review by Fedebravo12 — APPROVED** [review]
